src/autocomplete.js

```diff
@@ -288,15 +308,19 @@ class Autocomplete extends Component {
 
   /////////////////////////////// INPUT
 
+  getValue = itemValue => {
+    return itemValue ? this.props.getValue(itemValue) : ''
+  }
+
   getInputProps = ({onChange, onKeyDown, onBlur, ...rest} = {}) => {
-    const {value, selectedItem, isOpen} = this.state
+    const {isOpen} = this.state
     return {
       'data-autocomplete-input': true,
       role: 'combobox',
       'aria-autocomplete': 'list',
       'aria-expanded': isOpen,
       autoComplete: 'off',
-      value: (value === null ? this.getValue(selectedItem) : value) || '',
+      value: this.state.inputValue,
```

Comment:
Let's destructure this above. I like to destructure stuff at the top of the closure so it's easy to tell what this function uses.